tests/test_project/tests/test_password_history.py

```diff
@@ -100,3 +100,179 @@ def test_change_number_hasher_iterations(self):
                     user_config__iterations=HistoryVeryStrongHasher.iterations).count(),
                 1,
             )
+
+    @override_settings(AUTH_PASSWORD_VALIDATORS=[{
+        'NAME': 'django_password_validators.password_history.password_validation.UniquePasswordsValidator',
+        'OPTIONS': {
+            'lookup_range': 5
+        }
+    }])
+    def test_lookup_range_true_positive(self):
+        # We're considering the last 5 passwords and setting a valid password
+        self.create_user(1)
+        self.user_change_password(user_number=1, password_number=1)
+        self.user_change_password(user_number=1, password_number=2)
+        self.user_change_password(user_number=1, password_number=3)
+        self.assert_password_validation_False(user_number=1, password_number=1)
+        self.assert_password_validation_False(user_number=1, password_number=2)
+        self.assert_password_validation_False(user_number=1, password_number=3)
+        self.assert_password_validation_False(user_number=1, password_number=4)
+
+    @override_settings(AUTH_PASSWORD_VALIDATORS=[{
+        'NAME': 'django_password_validators.password_history.password_validation.UniquePasswordsValidator',
+        'OPTIONS': {
+            'lookup_range': 2
+        }
+    }])
+    def test_lookup_range_true_negative(self):
```

Comment:
@spokerman12, the name of this test case is unclear/ambiguous to me. Can you either explain what it means and how it matches what is being tested or if not, can you rename it?

Reply:
True negative implies that we're getting a real negative result, meaning the app is returning `False` or an error (password was used) and that's what we're expecting.

But I agree, it's not very clear. I'll rename it.